### PR TITLE
CrossSection & StructuralModel

### PR DESCRIPTION
**Pull request by JulienChampagnol**

### CODE REVIEW

File: assets/geode_objects.js [hunk 1]
```diff
@@ -15,10 +19,10 @@ const geode_objects = {
         tooltip: 'Graph',
         image: 'edged_curve.svg',
     },
-    // HybridSolid3D: {
-    //   tooltip: "HybridSolid3D",
-    //   image: "HybridSolid3D.svg",
-    // },
+    HybridSolid3D: {
```
Comment (BotellaA) [inline assets/geode_objects.js:22]: Is RegularGrid 2D and 3D missing?

Comment (JulienChampagnol) [reply assets/geode_objects.js:22]: Is the SVG file supposed to be the same for both regular grid 2D & 3D or not ?

Comment (BotellaA) [reply assets/geode_objects.js:22]: I would say different.

Comment (BotellaA) [reply assets/geode_objects.js:22]: let's use the same for now

**Review by BotellaA — APPROVED** [review]

**Pull request merged**